platform_versioned_docs/version-23.3.0/limits/limits.mdx

```diff
@@ -5,13 +5,13 @@ date: "21 Apr 2023"
 tags: [limits]
 ---
 
-[Seqera Cloud](https://tower.nf) elements and features have default limits per workspace and organization.
+Seqera Platform elements and features have default limits per workspace and organization.
 
 ### Workspaces
 
 | Description  | Value |
 | ------------ | ----- |
-| Active runs  | 5     |
+| Active runs  | Cloud: 5 Enterprise: Unlimited    |
```

Comment:
Sorry, @llewellyn-sl, I was wrong. I've re-checked and the default limit through License Manager is [5](https://github.com/seqeralabs/license-man/blob/26d503f6e34f78d22d9f366a881425a14ccca7b3/lm-backend/src/main/resources/db/migration/V04__add_default_quota_values.sql#L44-L45) too. Through config file, it is unlimited.

Reply:
@tcrespog Thanks. I've added an admonition to the page and added that config variable to the install guide.